superset/assets/src/explore/controls.jsx

```diff
@@ -1479,6 +1479,14 @@ export const controls = {
     description: t('Whether to display the trend line'),
   },
 
+  do_not_start_y_axis_at_zero: {
+    type: 'CheckboxControl',
+    label: t('Do not start y-axis at 0'),
```

Comment:
Can we use positive logic here. Instead of `do_not_start_y_axis_at_zero` can we make it `start_y_axis_at_zero`. Changing the name you can pull off the `!` https://github.com/apache/incubator-superset/pull/5552/files#diff-75b1598539b27c230eeef74daa95956cR165.